example_test.go

```diff
@@ -0,0 +1,60 @@
+package googlesearch
+
+import (
+	"context"
+	"fmt"
+	"strings"
+)
+
+func ExampleSearch() {
+
+	opt := SearchOptions{
+		CountryCode: "au",
+	}
+
+	serp, err := Search(context.Background(), "First Aid Course Australia Wide First Aid", opt)
```

Comment:
The example can be simplified to bare essentials by passing `nil` for context and removing context package (despite not being idiomatic go).

Reply:
Restored in dfdb3593e1e2388020736c362ba2bd21d4b6222e